docs/UserGuide.md

```diff
@@ -355,33 +351,28 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
-  * At least one leave record's field must be changed (`edit-leave 1` would be invalid as no leave record field are changed)
-  * Only fields specified will be modified; fields not specified in the command will not be modified. A field is specified by including its associated field prefix in the command (i.e. `title/` for title, `start/` for start date etc.)
-  * For title: if this field is included in the command, it cannot be left empty (i.e. `title/` is not allowed)
-  * For start and end: if these fields are included in the command, they must be in the format yyyy-mm-dd (like 2023-11-01 for 2023 November 1st). Additionally, the start date must be before or equals to the end date.
-  * For status: if this field is included in the command, it must be either APPROVED, PENDING or REJECTED
+3. Press "enter" on your keyboard and you should see the changes applied to the leave.
+* Examples:
 
-* **Expected output on failure:**
-  * `At least one field to edit must be provided` - if no fields are provided
-  * `The leave index provided is invalid` - if the index provided does not match with any leave record in the displayed leave list
-  * `FIELD must be of format...` - if at least one of the field inputs violates the field's format requirements. Follow the displayed error message to fix the format error.
-  * `end date cannot be before start date` - the end date provided is before the current or provided start date
+| Error message | Why it happens | Fix |
+|---------------|----------------|-----|
+| `Invalid command format!` | The command you input does not follow the specified format | Ensure the command you entered follows the following format: `edit-leav INDEX [title/TITLE] [start/START_DATE] [end/END_DATE] [d/DESCRIPTION] [s/STATUS]` where `INDEX` is the index of the leave in the list currently, `[title/TITLE]`, `[start/START_DATE]`, `[end/END_DATE]`, `[d/DESCRIPTION]`, `[s/STATUS]` are optional fields which require changing, replacing `TITLE` with the title of the leave, `START_DATE` with the leave's start date, `END_DATE` with the leave's end date, `DESCRIPTION` with the title's description and `STATUS` with the leave's status. |
+| `Date should be valid and in a format of yyyy-MM-dd`| The start or end date format does not conform to the program's standard. | Please update the date format to `yyyy-MM-dd` and reenter the comand. |
+| `The leave index provided is invalid` | The index specified does not refer to any leave | Double check if the index appears in the leave list. Alternatively, use [find-all-leave]() or any [find commands]() to locate the leave in the leave list. Afterwards, use the correct leave index in the `edit-leave` command. |
+| `At least one field to edit must be provided` | The command you input does not contain any fields to edit | Check if there is any input fields in the command inputted. An input like `edit-elave 1` is not accepted as there is no edits to be made. |
```

Comment:
would it be better to include one line "FIELD must be FORMAT" to account for things like TITLE including symbols, status not being APPROVED, PENDING or REJECTED? can see `edit` for more info